livre/eq/Config.cpp

```diff
@@ -131,6 +94,36 @@ const FrameData& Config::getFrameData() const
     return _impl->framedata;
 }
 
+::zerobuf::render::ImageJPEG& Config::getImageJPEG() const
+{
+#ifdef LIVRE_USE_ZEROEQ
+    return _impl->imageJPEG;
+#else
+    LBTHROW( std::runtime_error( "ZeroEq missing, impossible to send imageJPEG" ));
+#endif
+}
+
+bool Config::renderJPEG()
+{
+    _impl->imageJPEG.setData( std::vector< uint8_t >( ));
+    getFrameData().getFrameSettings().setGrabFrame( true );
+    frame();
+
+    while( true )
+    {
+        eq::EventICommand event = getNextEvent( 100000 );
```

Comment:
You wait indefinitely in 100s increments. You can as well wait indefinitely (default param) here.